docs/development/workflow/development_workflow.rst

```diff
@@ -370,15 +370,17 @@ Add a changelog entry
 *********************
 
 Add an entry to the file ``CHANGES.rst`` briefly describing the change you
-made. Include the pull request number if the change fixes an issue. An
+made. Include the pull request number, too at the end of the entry. An
```

Comment:
Nitpick: Is the comma necessary?

Reply:
I feel yes, but happy to be corrected or rewrite the sentence.

Reply:
Well, if you think it should belong, then I am okay with that. But looks like there is a conflict now and needs a rebase.

Reply:
Rebase done, but skipped CI this time, no need to waste kWs for non testable stuff.